### PR TITLE
Gce update

### PR DESCRIPTION
**Pull request by dnbit**

### CODE REVIEW

File: FinalProject/README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ https://developers.google.com/mobile-ads-sdk/docs/admob/android/quick-start
 You may need to download the Google Repository from the Extras section of the
 Android SDK Manager.
 
+You will also notice a folder called backend in the starter code. 
+It will be used at step 3 below, and you do not need to worry about it for now.
```
Comment (jwill) [inline FinalProject/README.md]: Should be "It will be used **in** step 3 below.

File: FinalProject/README.md [hunk 2]
```diff
@@ -69,8 +72,11 @@ http://developer.android.com/guide/components/intents-filters.html
 
 This next task will be pretty tricky. Instead of pulling jokes directly from
 our Java library, we'll set up a Google Cloud Endpoints development server,
-and pull our jokes from there. Follow the instructions in the following
-tutorial to add a Google Could Endpoints module to your project:
+and pull our jokes from there. 
+
+The following tutorial explains how to add and use a Google Cloud Endpoints module to
```
Comment (jwill) [inline FinalProject/README.md]: "Google Cloud Endpoints module **in** your project."

File: FinalProject/README.md [hunk 3]
```diff
@@ -69,8 +72,11 @@ http://developer.android.com/guide/components/intents-filters.html
 
 This next task will be pretty tricky. Instead of pulling jokes directly from
 our Java library, we'll set up a Google Cloud Endpoints development server,
-and pull our jokes from there. Follow the instructions in the following
-tutorial to add a Google Could Endpoints module to your project:
+and pull our jokes from there. 
+
+The following tutorial explains how to add and use a Google Cloud Endpoints module to 
+your project. The starter code already includes the module in the folder called backend.
+Therefore, you can skip the creation steps in the tutorial:
```
Comment (jwill) [inline FinalProject/README.md]: Might be good to tell them exactly what step to start at...looks like 1.1?

**Comment by dnbit** [comment]
Suggested fixes done :wink:

**Pull request merged**